### ISSUE TITLE
Implementar design da páginas de busca e navegação pelos Contratos

### ISSUE THREAD

**Issue by aivuk** [okfn-brasil/gastos_abertos_website#97]

**Comment by everton137** [comment]
@aivuk qual o design? Vale sempre colocar o link.

**Comment by aivuk** [comment]
@everton137  Isso foi o que pedi pra me ajudar a fazer, caçar os design espalhados no Google Drive e relacioná-los nas respectivas tarefas! :/

**Comment by everton137** [comment]
@aivuk verdade! É que achei que já tinha esse. Vou ajudar nisso e pedir para a @sibigauer me ajudar.

**Comment by sibigauer** [comment]
Busca nos contratos (ignorar versão velha de menu, etc): https://drive.google.com/file/d/0BwlESUhjI_hiSHFrSlpsU2VNbXM/view

**Comment by sibigauer** [comment]
Tabela: https://drive.google.com/file/d/0BwlESUhjI_hiR24zN2VZUS1HTjg/view

**Comment by sibigauer** [comment]
Gráfico ex horizontal: https://drive.google.com/file/d/0BwlESUhjI_hibTd4VnFGSkktZTg/view

**Comment by sibigauer** [comment]
Grafico ex vertical: https://drive.google.com/file/d/0BwlESUhjI_hicmtpd3pSSmxkeFE/view
